vendor/src/github.com/docker/notary/client/client.go

```diff
@@ -258,6 +258,22 @@ func (r *NotaryRepository) AddTarget(target *Target) error {
 	return cl.Close()
 }
 
+// RemoveTarget creates a new changelist entry to remove a target from the repository
+// when the changelist gets applied at publish time
+func (r *NotaryRepository) RemoveTarget(targetName string) error {
```

Comment:
This code is not currently being used by docker. It allows notary to remove remote tags.